tests/python/pants_test/scm/BUILD

```diff
@@ -3,7 +3,6 @@
 
 python_tests(
   name = 'test_git',
-  sources = ['test_git.py'],
```

Comment:
Is deleting this line still intentional or leftover from earlier changes?

Reply:
This was just a 1:1:1 change since this testing hasn't been updated in years, but can be reverted if it breaks something or is distracting.